python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -213,12 +213,12 @@ def __init__(
             simulation_mode=simulation_mode,
         )
 
-        # TODO DM-30686: remove this attribute and the code that uses it
-        # once ts_xml 9.1 is used everywhere.
-        self._electrical_has_motor_fields = hasattr(
+        # TODO DM-30952: remove this attribute and the code that uses it
+        # once ts_xml 9.2 is used everywhere.
+        self._actuators_has_timestamp_field = hasattr(
             self.tel_electrical.DataType(), "motorCurrent"
         )
-        if not self._electrical_has_motor_fields:
+        if not self._actuators_has_timestamp_field:
             self.log.warning(
```

Comment:
It seems this warning could be updated, with the new if condition?

Reply:
Thank you for catching this. In fact I messed up the test itself (I think I accidentally reverted some code while refactoring). So I have fixed the code and the warning message.